elasticsearch_metrics/metric.py

```diff
@@ -64,11 +72,19 @@ def get_index_template(cls):
     @classmethod
     def get_index_name(cls, date=None):
         date = date or timezone.now().date()
-        # TODO: Make dateformat configurable
-        dateformat = "%Y.%m.%d"
+        dateformat = settings.DATE_FORMAT
         date_formatted = date.strftime(dateformat)
         return "{}-{}".format(cls._template_name, date_formatted)
 
 
 class Metric(Document, BaseMetric):
     __doc__ = BaseMetric.__doc__
+
+    def save(self, using=None, index=None, validate=True, **kwargs):
+        self.timestamp = dt.datetime.now()
```

Comment:
This should use `timezone.now` so that it returns a timezone-aware datetime and respects Django's timezone settings.